core/js/src/Converters.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019-2021 JetBrains s.r.o.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime
+
+import kotlin.js.*
+
+/**
+ * Converts the [Instant] to an instance of [Date].
+ *
+ * The conversion is lossy: JS uses millisecond precision to represent dates, and [Instant] allows for nanosecond
+ * resolution.
+ */
+public fun Instant.toJSDate(): Date = Date(milliseconds = toEpochMilliseconds())
```

Comment:
Better to convert to `Double` (JS Number) explicitly to be safe from potential problems like https://youtrack.jetbrains.com/issue/KT-50657

Reply:
Fixed by using `toEpochMilliseconds().toDouble()`.